src/components/modals/email-confirm-modal/email-confirm-modal.js

```diff
@@ -0,0 +1,73 @@
+import { Button, Dialog, IconButton } from '@mui/material'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import CloseIcon from '@mui/icons-material/Close'
+import { style } from '../email-confirmation-modal.style'
+import { useState } from 'react'
+import img from '~/assets/img/email-confirmation-modals/icons8-check-circle-50.svg'
+import { styles } from '~/pages/error/styles/bad-request.styles'
+
+const EmailConfirmModal = () => {
+  const [open, setOpen] = useState(true)
```

Comment:
you don't need to create more modals. We already have a PopupDialog and need to pass content using a setModal

Reply:
used modal in GuestHomePage